bin/redis-commander.js

```diff
@@ -414,6 +481,67 @@ function createConnectionObjectFromArgs(argList) {
     if (argList['redis-tls']) {
       connObj.tls = {};
     }
+    if (argList['redis-tls'] &&
+      (argList['redis-tls-ca-cert-file'] || argList['redis-tls-ca-cert']
+        || argList['redis-tls-cert-file'] || argList['redis-tls-cert']
+        || argList['redis-tls-key-file'] || argList['redis-tls-key']
+        || argList['redis-tls-server-name'])) {
+      connObj.tls = {};
+      if (argList['redis-tls-ca-cert-file']) {
+        connObj.tls.ca = fs.readFileSync(argList['redis-tls-ca-cert-file']);
+      }
+      else if (argList['redis-tls-ca-cert']) {
+        connObj.tls.ca = argList['redis-tls-ca-cert'];
+      }
+      if (argList['redis-tls-cert-file']) {
+        connObj.tls.cert = fs.readFileSync(argList['redis-tls-cert-file']);
+      }
+      else if (argList['redis-tls-cert']) {
+        connObj.tls.cert = argList['redis-tls-cert'];
+      }
+      if (argList['redis-tls-key-file']) {
+        connObj.tls.key = fs.readFileSync(argList['redis-tls-key-file']);
+      }
+      else if (argList['redis-tls-key']) {
+        connObj.tls.key = argList['redis-tls-key'];
+      }
+      if (argList['redis-tls-server-name']) {
+        connObj.tls.servername = argList['redis-tls-server-name'];
+      }
+    }
+    if (argList['enable-sentinel-tls']) {
+      connObj.enableTLSForSentinelMode = true;
+    }
+    if (argList['sentinel-tls']) {
+      connObj.sentinelTLS = {};
+    }
+    if (argList['sentinel-tls'] &&
+      (argList['sentinel-tls-ca-cert-file'] || argList['sentinel-tls-ca-cert']
+        || argList['sentinel-tls-cert-file'] || argList['sentinel-tls-cert']
```

Comment:
Same as above for "redis-tls". Please no extra complex IF but another IF inside the `if (argList['sentinel-tls']) {`